symfony/test/functional/cypress/integration/orangehrm/admin/organization-general-info.spec.js

```diff
@@ -0,0 +1,211 @@
+import user from '../../../fixtures/admin-user.json';
+import charLength from '../../../fixtures/charLength.json';
+
+describe('Organization-General Info Page', function () {
+
+  beforeEach(() => {
+    cy.loginTo(
+      user.admin.userName,
+      user.admin.password,
+      '/admin/viewOrganizationGeneralInformation',
+    );
+  });
+
+  
+  describe('Verify Organization-General Info page properties', function () {
+    
+  it('Verify the title in Organization-General Info page', () => {
+    cy.visit('admin/viewOrganizationGeneralInformation');
+    cy.get('.orangehrm-main-title').should('include.text', 'General Information');
+
+  });
+
+  it('Verify the fields in Organization-General Info page', () => {
+    cy.get('.organization-name-container > .oxd-input-group > .oxd-input-group__label-wrapper > .oxd-label').should('include.text', 'Organization Name');
+    cy.get(':nth-child(1) > .oxd-grid-3 > :nth-child(2) > .oxd-input-group > .oxd-input-group__label-wrapper > .oxd-label').should('include.text', 'Number of Employees');
```

Comment:
Those repeating codes can abstract as `cy.get.shouldNthGridElement('include.text', 'Note', {nth-number})` and that can reuse other places as well. Finlay you can replace all lines as follows 
```
cy.get.shouldNthGridElement('include.text', 'Registration Number', 1);
cy.get.shouldNthGridElement('include.text', 'Tax ID, 2);
...
...

```